antlr-kotlin-gradle-plugin/src/main/java/com/strumenta/antlrkotlin/gradleplugin/AntlrKotlinTask.java

```diff
@@ -254,21 +242,24 @@ private void evaluate(AntlrResult result) {
         }
     }
 
-    /**
-     * Sets the source for this task. Delegates to {@link #setSource(Object)}.
-     * <p>
-     * If the source is of type {@link SourceDirectorySet}, then the relative path of each source grammar files
-     * is used to determine the relative output path of the generated source
-     * If the source is not of type {@link SourceDirectorySet}, then the generated source files end up
-     * flattened in the specified output directory.
-     *
-     * @param source The source.
-     * @since 4.0
-     */
-    @Override
-    public void setSource(FileTree source) {
-        setSource((Object) source);
-    }
+// Remove this: this is just simply unnecessary, it overwrites the default setter operator
```

Comment:
Please remove it then, instead of commenting it out

Reply:
Figured it had a purpose, so removed the comment